introducing-signal-store.md

```diff
@@ -0,0 +1,701 @@
+Welcome to **MiniRx Signal Store**, the new state management library from [MiniRx](https://mini-rx.io/).
+
+With the arrival of [Signals](https://angular.io/guide/signals) in Angular 16, new best practices are arising for state and events. In the future Signals and Observables will coexist in your application.
+
+Modern Angular needs modern state management which promotes **new Angular best practices** and **streamlines the usage of Signals and Observables**...
+
+MiniRx Signal Store is exactly that.
+
+### Modern state management with MiniRx Signal Store:
+
+* Signal Store is an **[Angular](https://angular.dev/)-only** state management library
```

Comment:
I'm not sure whether the "signal store" term as-is wouldn't be confused in terms of ngrx signal store vs minirx signal store. Not saying it will, bot not sure. At least I consider "signal store" somehow with ngrx 🤷‍♀️

Reply:
Yes, there could be indeed confusion.
But "MiniRx Signal Store" is the most logical name, considering the original "MiniRx Store" naming. "Signal" has to be in the name, I think.

MiniRx Store and MiniRx Signal Store have the same concept and lot of the APIs are identical. It would be weird to come up now with a totally different name.

"MiniRx" and "NgRx" is what makes the naming different enough, I hope :)